clang/include/clang/CConv/Constraints.h

```diff
@@ -617,11 +617,6 @@ class Constraints {
   ConstraintsGraph *PtrTypCG;
   std::map<std::string, ConstraintSet> constraintsByReason;
   ConstraintsEnv environment;
-  // Map of constraint variables, which are identified
-  // as itype pointers
-  // These should be the constraint variables of only
-  // function parameters or returns.
-  EnvironmentMap itypeConstraintVars;
```

Comment:
This was just unused before? Wacky.

Reply:
Yeah, only appears in that declaration and a getter. Not sure what it was used for.